posix.mak

```diff
@@ -372,9 +376,18 @@ endif
 # macro that returns the module name given the src path
 moduleName=$(subst /,.,$(1))
 
-unittest/%.d : $(ROOT)/unittest/test_runner
+# target for batch unittests (using shared phobos library and test_runner)
+unittest/%.run : $(ROOT)/unittest/test_runner
 	$(QUIET)$(RUN) $< $(call moduleName,$*)
 
+# target for quickly running a single unittest (using static phobos library)
+%.test : %.d $(LIB)
+	$(DMD) $(DFLAGS) -main -unittest $(LIB) -defaultlib= -debuglib= -L-lcurl -run $<
```

Comment:
I wonder why you try so hard to make tests re-runnable, what's the purpose of running the same test twice?